L1Trigger/TrackerTFP/interface/DataFormats.h

```diff
@@ -0,0 +1,688 @@
+#ifndef L1Trigger_TrackerTFP_DataFormats_h
+#define L1Trigger_TrackerTFP_DataFormats_h
+
+/*----------------------------------------------------------------------
+Classes to calculate and provide dataformats used by Track Trigger emulator
+enabling automated conversions from frames to stubs/tracks and vice versa
+----------------------------------------------------------------------*/
+
+#include "FWCore/Framework/interface/data_default_record_trait.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormatsRcd.h"
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "DataFormats/L1TrackTrigger/interface/TTBV.h"
+
+#include <vector>
+#include <cmath>
+#include <initializer_list>
+#include <tuple>
+#include <iostream>
+#include <string>
+
+namespace trackerTFP {
+
+  // track trigger processes
+  enum class Process { begin, fe = begin, dtc, pp, gp, ht, mht, sf, kfin, kf, dr, end, x };
```

Comment:
Process PP, sitting between DTC & GP, is mentioned in several places. Please add a comment explaining what it is.

Reply:
where shall I put this comment?

Reply:
Possibly it belongs in README.md?